docs/api.md

```diff
@@ -1314,8 +1365,129 @@ puppeteer.launch().then(async browser => {
 });
 ```
 
+### class: ExecutionContext
+
+The class represents a context for javascript execution. Examples of javascript contexts are:
+- each frame has a separate execution context
```

Comment:
Link to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe

Reply:
Done.